### ISSUE TITLE
Help choosing recipe

### ISSUE THREAD

**Issue by muntasir2000** [espnet/espnet#3384]
Hi, I have 250 hours of data and a single 24GB 3090 GPU. Which recipe should be a good starting point for me to train in a reasonable amount of time (4-5 days)? Thanks in advance.

**Comment by sw005320** [comment]
Which language?
Anyway, librispeech https://github.com/espnet/espnet/tree/master/egs2/librispeech/asr1 would be good in terms of the state-of-the-art recipe or https://github.com/espnet/espnet/tree/master/egs2/swbd/asr1 would be a good in terms of the similar amount of training data (300 hours).

I believe you can train a 250 hours of data model with a single 24GB 3090 GPU in 4-5 days.
I trained a swbd model (300 hours) with Titan RTX (24GB), and it took 3-4 days.

**Comment by muntasir2000** [comment]
Thanks for your response. I'm training ASR for Bengali language.

I started training with the Switchboard recipe. But one problem I'm facing right now is, I am facing GPU OOM errors. I tried reducing batch-bins from 75000000 (originally mentioned in the recipe for 32GB GPU) to 8000000. But still facing OOM. My GPU memory is 24GB, which is not that less from the original 32GB of the recipe. 

I also tried setting `use_amps: true` in the config file. But the issue remains.
